OpenKJ/OpenKJ.pro

```diff
@@ -359,6 +360,7 @@ HEADERS  += mainwindow.h \
     karaokefileinfo.h \
     dlgeditsong.h \
     soundfxbutton.h \
+    runguard/runguard.h
```

Comment:
This line is missing a trailing \
This is required to tell qmake that the list of files continues on the next line